bundle/Installer/NetgenRemoteMediaSiteInstaller.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteInstallerBundle\Installer;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\API\Repository\Values\ContentType\ContentTypeDraft;
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition;
+use Throwable;
+
+class NetgenRemoteMediaSiteInstaller extends NetgenSiteInstaller
+{
+    /**
+     * @var \eZ\Publish\API\Repository\ContentTypeService
+     */
+    private $contentTypeService;
```

Comment:
Missing property typehints

Reply:
Added in b345e69d55487f3449b048cb589da0e207e5c358